python/pyspark/ml/regression.py

```diff
@@ -708,7 +710,7 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeModel(JavaModel):
+class DecisionTreeModel(JavaModel, JavaPredictionModel):
```

Comment:
That seems reasonable, I've got some less direct code things I need to catch up on for the next few days, but if no one picks up the regression stuff in the meantime I'll roll that into this PR.